packages/desktop-client/src/components/FatalError.tsx

```diff
@@ -157,7 +173,7 @@ export function FatalError({ buttonText, error }: FatalErrorProps) {
   const showSimpleRender = 'type' in error && error.type === 'app-init-failure';
 
   return (
-    <Modal isCurrent={true} CloseButton={undefined} title="Fatal Error">
+    <Modal isCurrent title="Fatal Error">
```

Comment:
Maybe we can accept a custom title here and pass in "Loading Error" for lazy loading errors.

Reply:
Sure: https://github.com/actualbudget/actual/pull/2657/commits/6dd311f12fcc9e5d166e3f1d35d10c00ea590a19

It required a small amount of refactoring though. :)